extension/Hello/Tasks/ByeWorldTask.yaml

```diff
@@ -0,0 +1,20 @@
+---
+id: 'bye:world'
+short_description: 'Bye bye'
+help: 'Will greet the one using it'
+stage: bye
+version: 1.0.0
+deprecated: false
+successor: 'hello:php'
+command: '/bin/echo "bye %world%"'
+type: local_cli
+tags: ['bye', 'world']
```

Comment:
we already have tags in task set. in this case if we provide tags from task set we need overwrite these ones for task set

Reply:
Got it, I'll implement overriding of tags